### PR TITLE
Implement EIP-1283 reenable transition, EIP-1706 and EIP-2200

### PR DESCRIPTION
**Pull request by sorpaas**
Reentry protection for EIP-1283, described in https://github.com/ethereum/EIPs/pull/1706. With `eip1884_transition` this also implements EIP-2200.

Note that the correct way to enable this on Ethereum mainnet would be to set both `eip1283_reenable_transition` and `eip1706_transition`.

### CODE REVIEW

**Comment by 5chdn** [comment]
Putting this on ice because, ideally, we want to only implement EIPs that are accepted or we are really certain we want to further diverge from mainnet. I'd say, let's disable 1283 on kovan for now.

**Comment by 5chdn** [comment]
I'm down to accept this.

**Comment by soc1c** [comment]
Let's revisit this once we actually need it.

**Comment by dvdplm** [comment]
@sorpaas time to revive this?

**Comment by sorpaas** [comment]
Will get on to it.

**Comment by dvdplm** [comment]
@sorpaas re-labeling this "in-progress" for the time being, ok?

**Comment by sorpaas** [comment]
@dvdplm Should be ready for review again!

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+_If you think that your issue is an exploitable security vulnerability, please mail your bugreport to security@parity.io instead; your submission might be eligible for our Bug Bounty._
+_You can find mode info on the reporting process in [SECURITY.md](https://github.com/paritytech/parity-ethereum/blob/master/SECURITY.md)_
+
+
 - **Parity Ethereum version**: 0.0.0
 - **Operating system**: Windows / MacOS / Linux
 - **Installation**: homebrew / one-line installer / built from source
 - **Fully synchronized**: no / yes
-- **Network**: ethereum / ropsten / kovan / ...
+- **Network**: ethereum / ropsten / goerli / ...
```
Comment (soc1c) [inline .github/ISSUE_TEMPLATE.md]: :hugs:

**Comment by ordian** [comment]
was `schedule.eip1706 = block_number >= self.eip1706_transition;` lost in merge?

**Comment by sorpaas** [comment]
@ordian You're right. My fault.

**Pull request merged**